tests/ui/range.rs

```diff
@@ -14,3 +14,40 @@ fn no_panic_with_fake_range_types() {
 
     let _ = Range { foo: 0 };
 }
+
+#[warn(clippy::manual_range_contains)]
```

Comment:
Can you move those to a new test file and add `// run-rustfix` there please?

Reply:
Sure thing!